roles/1-prep/defaults/main.yml

```diff
@@ -46,12 +46,14 @@ ap_device: "none"
 device_gw2: ""
 
 # WiFi
-ssid: XSCE
+ssid: IIAB
```

Comment:
@georgejhunt what is this "ssid: IIAB" used for?

Also is there a reason it's different from the following that we have in local_vars & default_vars ?

`host_ssid: "Internet in a Box"`

Reply:
It's a starting point default variable, to guard against users removing SSID from *_vars and causing ansible to fail